lib/foods.js

```diff
@@ -47,9 +52,25 @@ function tableFilter() {
 $('document').ready(function() {
   $("#add-food").on('click', function() {
     checkInputs();
-    loadButton();
+    createDeleteButton();
+    $('td').on('click', function() {
+      var $this = $(this);
+      var $input = $('<input>', {
+        value: $this.text(),
+        type: 'text',
+        blur: function() {
+          $this.text(this.value);
+        },
+        keyup: function(e) {
+          if (e.which === 13) $input.blur();
+        }
+      }).appendTo( $this.empty() ).focus();
```

Comment:
@lucyconklin For the record based on our joint review I am clear on the purpose of `this.empty()` => it is needed to empty out the `td` so it would only contain the updated value and not a mix of old and updated value. Thank you for your help to clarify this! 👍